### PR TITLE
SRCH-4631 Re-enable redesign=true URL parameter

### PR DESCRIPTION
**Pull request by ddzz**
## Summary
SRCH-4631 Re-enable redesign=true URL parameter
 
### Checklist
Please ensure you have addressed all concerns below before marking a PR "ready for review" or before requesting a re-review. If you cannot complete an item below, replace the checkbox with the ⚠️ `:warning:` emoji and explain why the step was not completed.
 
#### Functionality Checks
 
- [x] You have merged the latest changes from the target branch (usually `main`) into your branch.
  
- [x] Your primary commit message is of the format **SRCH-#### \<description\>** matching the associated Jira ticket.

- [x] PR title is either of the format **SRCH-#### \<description\>** matching the associated Jira ticket (i.e. "SRCH-123 implement feature X"), or **Release - SRCH-####, SRCH-####, SRCH-####** matching the Jira ticket numbers in the release.
 
- [x] Automated checks pass. If Code Climate checks do not pass, explain reason for failures:
 
#### Process Checks

- [x] You have specified at least one "Reviewer".

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -127,15 +127,24 @@ def set_search_params
     @search_params = ActiveSupport::HashWithIndifferentAccess.new(query: @search.query, affiliate: @affiliate.name)
     @search_params[:sitelimit] = permitted_params[:sitelimit] if permitted_params[:sitelimit].present?
     @search_params[:dc] = permitted_params[:dc] if permitted_params[:dc].present?
+    @search_params[:redesign] = permitted_params[:redesign] if permitted_params[:redesign].present?
```
Comment (ddzz) [inline app/controllers/application_controller.rb:130]: This line and L165 are the only changes in this PR. The rest are Rubocop fixes.

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -153,6 +162,6 @@ def append_info_to_payload(payload)
   end
 
   def redesign?
-    @affiliate.use_redesigned_results_page
+    @affiliate.use_redesigned_results_page or permitted_params[:redesign] == 'true'
```
Comment (stevenbarragan) [inline app/controllers/application_controller.rb:165]: small improvement but what do you think about using `permitted_params[:redesign].present?` here as well, that way we can set the variable to anything to enable it.

**Review by stevenbarragan — APPROVED** [review]
I caught an small improvement but it LGTM

File: app/controllers/application_controller.rb (hunk 2 above)
Comment (ddzz) [reply app/controllers/application_controller.rb:165]: We have some tests using the hard-coded `true`. This PR would need more changes to use `#present?`.

**Pull request merged**